Detectors/TPC/qc/src/Tracks.cxx

```diff
@@ -56,7 +56,17 @@ void Tracks::resetHistograms()
 }
 
 //______________________________________________________________________________
-bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
+bool Tracks::processTracks(const gsl::span<const o2::tpc::TrackTPC>& tracks)
+{
+  for (auto const& track : tracks) {
+    Tracks::processOneTrack(track);
```

Comment:
I think you don't need the scope to call the function, just
```c++
processTrack(track);
```
after having renamed the function back to `processTrack`.

Please put this function even as inline in the header.

Reply:
Ok, I will rename the function to `processTrack`. I thought, for readability it would be good, not to have to almost identical functions. Maybe one could instead just call the new function `processAllTracks`?
I think, putting the function as inline will not work, because it contains a for loop, so I would keep it here.

Reply:
I think the function parameter is enough to understand what input is supposed.
I would stay with `processTrack` and `processTracks`.

Reply:
Or one could even make it shorter and simply call both functions `process` and one can easily see from the function parameter what is meant.

Reply:
Then I would keep both `processTrack` and `processTracks`, as otherwise again I would have to change the root macro and the QC workflow, which we just tried to avoid.

Reply:
ok